atc/metric/emitter/newrelic.go

```diff
@@ -1,41 +1,74 @@
 package emitter
 
 import (
+	"bufio"
 	"bytes"
+	"compress/gzip"
 	"encoding/json"
 	"fmt"
+	"github.com/concourse/flag"
```

Comment:
nit: we usually separate these imports from stdlib - e.g.: https://github.com/concourse/concourse/blob/7a6048b7d6a9124c891b58d9560a9af8be80edaf/atc/db/build_factory.go#L3-L11